code/game/say.dm

```diff
@@ -44,7 +44,7 @@ GLOBAL_LIST_INIT(freqtospan, list(
 /atom/movable/proc/get_spans()
 	return list()
 
-/atom/movable/proc/compose_message(atom/movable/speaker, datum/language/message_language, raw_message, radio_freq, list/spans, message_mode)
+/atom/movable/proc/compose_message(atom/movable/speaker, datum/language/message_language, raw_message, radio_freq, list/spans, message_mode, face_name = FALSE)
```

Comment:
Make sure you update all overrides of compose_message

Reply:
There are none!